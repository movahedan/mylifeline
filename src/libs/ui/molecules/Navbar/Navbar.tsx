import Link from "next/link";

import { routes } from "@mylifeline/constants";
import { classNames } from "@mylifeline/utilities";

import type { CSSProperties } from "react";

export type NavbarProps = {
  className?: string;
  style?: CSSProperties;
};

export const Navbar = ({ className, style }: NavbarProps) => {
  return (
    <nav
      style={style}
      className={classNames([
        "navbar",
        "flex justify-center items-center",
        className,
      ])}
    >
      <ul className="space-x-32">
        <Link href={routes.index}>Home</Link>
        <Link href={routes.goals}>Goals</Link>
      </ul>
    </nav>
  );
};
