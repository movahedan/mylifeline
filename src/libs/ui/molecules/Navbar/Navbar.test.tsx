import { render } from "@testing-library/react";

import { Navbar } from "./Navbar";

describe("Navbar", () => {
  test("snapshot test", () => {
    const { container } = render(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
