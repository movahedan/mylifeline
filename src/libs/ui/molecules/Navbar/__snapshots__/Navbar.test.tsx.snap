// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot test 1`] = `
<div>
  <nav
    class="navbar flex justify-center items-center"
  >
    <ul
      class="space-x-32"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/goals"
      >
        Goals
      </a>
    </ul>
  </nav>
</div>
`;
